src/ray/gcs/gcs_client/test/service_based_gcs_client_test.cc

```diff
@@ -28,9 +28,7 @@ class ServiceBasedGcsClientTest : public ::testing::Test {
  public:
   ServiceBasedGcsClientTest() {
     RayConfig::instance().initialize(
-        {{"ping_gcs_rpc_server_max_retries", std::to_string(60)},
-         {"maximum_gcs_destroyed_actor_cached_count", std::to_string(10)},
-         {"maximum_gcs_dead_node_cached_count", std::to_string(10)}});
+        "ping_gcs_rpc_server_max_retries,60;maximum_gcs_destroyed_actor_cached_count,10;maximum_gcs_dead_node_cached_count,10");
```

Comment:
Will colon be better than comma?

Reply:
I'm not going to update the format of serialized config in this PR.